privaterelay/settings.py

```diff
@@ -40,7 +40,7 @@
     # https://github.com/jazzband/django-silk
     import silk
 
-    assert silk  # Suppress "imported but unused" warning
+    assert silk  # Suppress "imported but unused" warning # noqa S101
```

Comment:
Nit: use `# noqa F401` on the import, drop the `assert`

Reply:
Thanks. I like the `noqa F401` on the single line rather than `noqa S101` on the additional line.